neuralcoref/train/conllparser.py

```diff
@@ -595,20 +595,24 @@ def read_corpus(self, data_path, debug=False):
                                       embedding_extractor=self.embed_extractor,
                                       conll=CONLL_GENRES[name[:2]]))
         print("🌋 Loading spacy model")
-        model_options = ['en_core_web_lg', 'en_core_web_md', 'en_core_web_sm', 'en']
-        model = None
-        for model_option in model_options:
-            if not model:
-                try:
-                    spacy.info(model_option)
-                    model = model_option
-                    print("Loaded model", model_option)
-                except:
-                    print("Could not detect model", model_option)
-        if not model:
-            print("Could not detect any suitable English model")
-            return
 
+        if model is None:
+            model_options = ['en_core_web_lg', 'en_core_web_md', 'en_core_web_sm', 'en']
+            model = None  # this declaration is redundant
```

Comment:
if it's redundant (and it is in the current version of the code), feel free to just remove it ;-)

Reply:
Removed it